cosmos/dbt/graph.py

```diff
@@ -597,11 +647,12 @@ def load_via_dbt_ls_without_cache(self) -> None:
                     self.profile_config.target_name,
                 ]
 
-                self.log_dir = Path(env.get(DBT_LOG_PATH_ENVVAR) or tmpdir_path / DBT_LOG_DIR_NAME)
                 self.target_dir = Path(env.get(DBT_TARGET_PATH_ENVVAR) or tmpdir_path / DBT_TARGET_DIR_NAME)
                 env[DBT_LOG_PATH_ENVVAR] = str(self.log_dir)
```

Comment:
Will `self.log_dir` still be `None` at this point of time? And would we be altering the env var to None always in that case? Is that our desired behavior?

With this, in the below line it would always resort & set `self.log_dir` to `tmpdir_path/DBT_LOG_DIR_NAME`, no?

Reply:
Nice catch, @pankajkoti , I fixed this in 7db1de03833ee44e434a91f023e6105d6de0582a